geopandas/base.py

```diff
@@ -94,7 +94,31 @@ class GeoPandasBase(object):
     @property
     def area(self):
         """Returns a ``Series`` containing the area of each geometry in the
-        ``GeoSeries``."""
+        ``GeoSeries``.
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import Polygon
+        >>> s = geopandas.GeoSeries(
+                [
```

Comment:
```suggestion
        ...     [
```

(is in principle needed for properly formatted docstring examples, and the same for the lines below)

Reply:
With `...`, it is properly formatted (i.e. mimics the output), but you can't copy&paste the snippet. Without them, you can directly copy and paste it and it will work.

But we can add `sphinx-copybutton` (https://sphinx-copybutton.readthedocs.io/en/latest/) to go around it.

Reply:
Yes, I think solving this at the sphinx documentation level is the way to go. 

Alternative to sphinx-copybutton are https://github.com/jurasofish/sphinx-toggleprompt (which is what the Python docs do, but then put in an extension)